docs/life-of-a-feature.md

```diff
@@ -0,0 +1,29 @@
+# Life of a Feature
```

Comment:
https://github.com/bytebase/bytebase/blob/56a9ed93a98bb94ad02fe9197507b277f8b31f80/docs/life-of-a-feature.md